pkg/controller/pods.go

```diff
@@ -202,7 +239,37 @@ func (cont *AciController) podDeleted(obj interface{}) {
 
 	cont.indexMutex.Lock()
 	cont.removePodFromNode(pod.Spec.NodeName, podkey)
+	nmport := false
+	for _, ctr := range pod.Spec.Containers {
+		for _, ctrportspec := range ctr.Ports {
+			if ctrportspec.Name != "" {
+				portmap, ok := cont.ctrPortNameMap[ctrportspec.Name]
+				if ok {
+					key := portProto(&ctrportspec.Protocol) + "-" + strconv.Itoa(int(ctrportspec.ContainerPort))
+					if portmap[key] == 1 {
+						delete(cont.ctrPortNameMap, ctrportspec.Name)
```

Comment:
Could you get a port name change as part of a pod update as well?

Reply:
no, i am not able to edit the port name in pod spec